ergo/metaculus.py

```diff
@@ -161,7 +230,13 @@ def submit(self, loc, scale):
             },
             data=json.dumps(prediction_data)
         )
-        r.raise_for_status()
+        try:
+            r.raise_for_status()
+
+        except requests.exceptions.HTTPError as e:
+            e.args = (str(
+                e.args), f"request body: {e.request.body}", f"response json: {e.response.json()}")
```

Comment:
It's really handy to have the request body and response json in order to debug these exceptions

there might be a better way to get them then assigning to the exception's args, though

Reply:
Why are you converting `e.args`, `e.request.body` and `e.response.json()` to strings?